client/src/components/Learn/Learn.jsx

```diff
@@ -1,85 +1,50 @@
 import React from 'react';
```

Comment:
changes made here are a big departure from the core design philosophy of the website. It looks nice but strays away from minimalism easy consumption (this redesign requires a lot of scrolling between sections). I know the issue for this page was only "Refactor Learn page" but what I meant by that was mostly bringing it inline with the design of the other pages, namely 'About', some of which has already been implemented on the r/newgame branch. If you have any question as to the direction of the design, check out https://lichess.org/. It should be clear enough but I'm taking most of the design inspiration from that site fairly directly, as I see it as the best chess site out right now.